resources/services/kinesis/streams.go

```diff
@@ -0,0 +1,189 @@
+package kinesis
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/kinesis"
+	"github.com/aws/aws-sdk-go-v2/service/kinesis/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Streams() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_kinesis_streams",
+		Description:  "Represents the output for DescribeStreamSummary",
+		Resolver:     fetchKinesisStreams,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("kinesis"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("StreamARN"),
+			},
+			{
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: ResolveKinesisStreamTags,
+			},
+			{
+				Name:        "open_shard_count",
+				Description: "The number of open shards in the stream.  This member is required.",
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "retention_period_hours",
+				Description: "The current retention period, in hours.  This member is required.",
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "stream_arn",
+				Description: "The Amazon Resource Name (ARN) for the stream being described.  This member is required.",
```

Comment:
remove `This member is required.` here and in other columns. It can be done via cq-gen

Reply:
fixed